spec/pagerduty/http_transport_spec.rb

```diff
@@ -10,27 +10,28 @@
   Given(:post) { double.as_null_object }
   Given { allow(Net::HTTP::Post).to receive(:new).and_return(post) }
 
-  describe "::send_payload" do
-    Given(:payload) { {
-      event_type: "trigger",
-      service_key: "test-srvc-key",
-      description: "test-desc",
-      details: { key: "value" },
-    } }
+  describe '::send_payload' do
+    Given(:payload) do
+      {
+        event_type: 'trigger',
+        service_key: 'test-srvc-key',
+        description: 'test-desc',
+        details: { key: 'value' }
+      }
+    end
 
     When(:response) { http_transport.send_payload(payload) }
 
-    describe "provides the correct request" do
-      Then {
+    describe 'provides the correct request' do
+      Then do
```

Comment:
RSpec-Given traditionally uses `{ }` for multiline blocks because `do end` interferes with the language of the specification. I'd like to keep the `{ }` in these cases.

Can we limit this check to the production code?